src/main/java/gov/usgs/locaux/LocUtil.java

```diff
@@ -152,30 +152,29 @@
   public static final double DEG2KM = 6371d * Math.PI / 180d;
 
   /** An int constant representing the maximum number of iteration stages to attempt. */
-  public static final int STAGELIMIT = 2;
+  /** JMP 1/13/2022 Added a third stage due to large number of events that did not converge */
+  public static final int STAGELIMIT = 3;
 
   /** A double constant representing the initial step length to start each iteration loop with. */
   public static final double INITIALSTEPLEN = 50d;
 
   /** An array of int constants representing the maximum number of iterations for each stage. */
-  // public static final int[] ITERATIONSTAGELIMITS = {15, 20, 20, 20, 20, 20, 20, 20, 20, 20};
-  public static final int[] ITERATIONSTAGELIMITS = {
-    15, 20
-  }; // Collapse to with and without decorrelation sub-loops 9/16/19.
+  /** JMP 1/13/2022 Added a third stage due to large number of events that did not converge */
+  public static final int[] ITERATIONSTAGELIMITS = {15, 20, 30};
 
   /**
    * An array of double constants representing the convergence criteria in kilometers for each
    * stage.
    */
-  // public static final double[] CONVERGENCESTAGELIMITS = {1d, 0.1d, 0.1d, 0.1d, 0.1d, 0.1d, 0.1d,
-  // 		0.1d, 0.1d, 0.1d};
-  public static final double[] CONVERGENCESTAGELIMITS = {1d, 0.1d};
+  /** JMP 1/13/2022 Added a third stage due to large number of events that did not converge */
+  public static final double[] CONVERGENCESTAGELIMITS = {1d, 0.1d, 0.1d};
```

Comment:
probably obvious to you, but again, don't understand why the 3rd repeats the second, I see the old comments that had many of these in the past...

Reply:
These are the limits before the stage declares the location converged, the smaller you go, the harder it is to converge. In this case, we just want to try for more iterations with the same convergence limits as in the second stage, so we just duplicate the limit. (also that's what ray did in the old stage configuration so I followed his lead)